### PR TITLE
[SHELL32] ro-RO.rc Improvement

### PR DESCRIPTION
**Pull request by JoachimHenze**
Addendum to https://github.com/reactos/reactos/pull/6517 (the rules) and
Addendum to https://github.com/reactos/reactos/pull/6657 (Andreis recent shell32 PR)

Following the rule:
1. We use only the correct Romanian language diacritics.
     This means using lowercase ţ instead of ț.
     This means using lowercase ş instead of ș.
     This means using uppercase Ş instead of Ș.
     Some of XP/2k3sp2 fonts do lack ț and ș and Ș.

2. Avoid the word "calculator" for the shutdown/logoff prompt (same as Andrei Miloiu did before in his PR at many locations).
     Using "sistem" is fine, and using "computer" is fine.

3. Harmonize some strings used for some FILEOPENDLG_FILTER.

AndreiMiloiu please review this PR!

## TODO

- [] In a future PR we will have to do the same character replacements for the Shell folder path default values. This has to be done in sync with dll/win32/userenv/lang therefore I left them out here for now, for easier reviewability of those more delicate parts.

### CODE REVIEW

**Comment by JoachimHenze** [comment]
@AndreiMiloiu can you please review this PR?

File: dll/win32/shell32/lang/ro-RO.rc [hunk 1]
```diff
@@ -736,11 +736,11 @@ STYLE DS_SHELLFONT | DS_MODALFRAME | WS_POPUP | WS_CAPTION | WS_SYSMENU
 FONT 8, "MS Shell Dlg"
 BEGIN
     ICON 0, IDC_LINK_PROBLEM_ICON, 5, 5, 0, 0
-    LTEXT "Elementul '%s' la care această scurtătură face referire a fost schimbată sau mutată, așa că scurtătura nu va mai funcționa corect.", IDC_LINK_PROBLEM_LABEL1, 35, 5, 210, 35
-    LTEXT "Cea mai apropiată potrivire, bazată pe nume, tip, mărime și dată:", IDC_LINK_PROBLEM_LABEL2, 35, 45, 210, 35
-    LTEXT "Reparați scurtătura pentru a atinge ținta sau doar vreți să o ștergeți?", -1, 35, 85, 210, 30
+    LTEXT "Elementul '%s' la care această scurtătură face referire a fost schimbată sau mutată, aşa că scurtătura nu va mai funcţiona corect.", IDC_LINK_PROBLEM_LABEL1, 35, 5, 210, 35
+    LTEXT "Cea mai apropiată potrivire, bazată pe nume, tip, mărime şi dată:", IDC_LINK_PROBLEM_LABEL2, 35, 45, 210, 35
+    LTEXT "Reparaţii scurtătura pentru a atinge ţinta sau doar vreţi să o ştergeţi?", -1, 35, 85, 210, 30
```
Comment (JoachimHenze) [inline dll/win32/shell32/lang/ro-RO.rc:741]: notice the added i -> ii. Is that correct?

Comment (AndreiMiloiu) [reply dll/win32/shell32/lang/ro-RO.rc:741]: No, it isn't. It must be only a single i.

Comment (JoachimHenze) [reply dll/win32/shell32/lang/ro-RO.rc:741]: Thank you!
```suggestion
    LTEXT "Reparaţi scurtătura pentru a atinge ţinta sau doar vreţi să o ştergeţi?", -1, 35, 85, 210, 30
```

Comment (AndreiMiloiu) [reply dll/win32/shell32/lang/ro-RO.rc]: You're welcome!

**Comment by AndreiMiloiu** [comment]
At line 926 it must be either Orice fișier or Toate fișierele. You changed Orice with Toate without adaptating the noun to the article which is wrong.

**Comment by AndreiMiloiu** [comment]
The same for line 927. Everywhere where it writes Toate fișier must be changed either by my first or my second suggestion.

**Review by AndreiMiloiu — CHANGES_REQUESTED** [review]
Toate fișier must be changed everywhere with either:

Orice fișier (like in MSDN)

or

Toate fișierele

**Comment by JoachimHenze** [comment]
> Toate fișier must be changed everywhere with either:
> 
> Orice fișier (like in MSDN)
> 
> or
> 
> Toate fișierele

Oof, I just copied here, what you did before in your PR to another location. *eyeroll*

**Comment by AndreiMiloiu** [comment]
> > Toate fișier must be changed everywhere with either:
> > Orice fișier (like in MSDN)
> > or
> > Toate fișierele
> 
> Oof, I just copied here, what you did before in your PR to another location. _eyeroll_

Yes, you're right. Sorry for that. I didn't notice the mistakes before.

**Comment by AndreiMiloiu** [comment]
Thank you @JoachimHenze!

**Comment by AndreiMiloiu** [comment]
Now I discover other mistakes I made. For lines 846 and 848 please replace the word "sistemul" with the word "computer" (yes, without any article) in order to be the same everywhere.

**Comment by AndreiMiloiu** [comment]
For lines 846 and 848: please, leave "Reporniți sistemul?" and "Închideți calculatorul?", respectively as it was before. Please, revert these changes.

**Comment by AndreiMiloiu** [comment]
@JoachimHenze I came back with some reviews related to what I told you before. For the lines 846 and 848 there should be either Reporniți sistemul? and Închideți sistemul?, respectively or Repornire sistem? and Închidere sistem?, respectively.

**Comment by JoachimHenze** [comment]
> @JoachimHenze for the lines 846 and 848 there should be either Reporniți sistemul? and Închideți sistemul?, respectively or Repornire sistem? and Închidere sistem?, respectively. Închidere sistemul? and Repornire sistemul? are wrong.

Thank you for the tip. I will go with `Repornire sistem?` and `Închidere sistem?` then.

**Comment by AndreiMiloiu** [comment]
> > @JoachimHenze for the lines 846 and 848 there should be either Reporniți sistemul? and Închideți sistemul?, respectively or Repornire sistem? and Închidere sistem?, respectively. Închidere sistemul? and Repornire sistemul? are wrong.
> 
> Thank you for the tip. I will go with `Repornire sistem?` and `Închidere sistem?` then.

Yes

**Pull request merged**